routes/api/comments.js

```diff
@@ -0,0 +1,63 @@
+//2 rts
+const express = require('express');
+const {asyncHandler} = require('../utils');
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+const router = express.Router();
+const { sequelize, CommentLike } = require("../../db/models");
+
+/* Like a comment */
+router.post('/comments/:id/likes', restoreUser, asyncHandler(async(req, res) => {
+
+    if (!res.locals.authenticated || res.locals.user.id !== like.userId) {
+        /* Return error 401, if
+               user is not authenticated, or
+               user is not liker */
+        return next(createError(401)); 
```

Comment:
The code does not have access to a `next` in this scope

Reply:
`next` can now be accessed.